spec/views/users/show.html.erb_spec.rb

```diff
@@ -29,7 +29,11 @@
 
   context "with trophy" do
     let(:trophy_presenter) { Sufia::TrophyPresenter.new(solr_document) }
-    let(:solr_document) { SolrDocument.new(id: 'abc123', has_model_ssim: 'GenericWork') }
+    let(:solr_document) do
+      SolrDocument.new(id: 'abc123',
+                       has_model_ssim: 'GenericWork',
+                       thumbnail_path_ss: '/foo/bar.png')
```

Comment:
Yeah, otherwise the view will attempt a `image_tag(nil)`, which is an error in Rails 5